src/Features/Core/Portable/Workspace/MiscellaneousFileUtilities.cs

```diff
@@ -18,10 +19,12 @@ internal static class MiscellaneousFileUtilities
             string filePath,
             TextLoader textLoader,
             LanguageInformation languageInformation,
+            SourceHashAlgorithm checksumAlgorithm,
```

Comment:
Just thinking ahead, is there a need for misc files to have a checksum algorithm for E&C without a project or something?